docs/README.md

```diff
@@ -4,24 +4,32 @@
 
 We welcome contributions on our documentation as well under the same code of conduct as the rest of the project.
 
-To build & locally view the docs you can follow [these steps](https://www.mkdocs.org/#building-the-site):
+To edit and locally view the docs you can follow [these steps](https://www.mkdocs.org/#building-the-site):
 
-From the root:
+## One-time setup:
 
-1. pip install --upgrade -r requirements.txt
-2. mkdocs build --clean
-3. mkdocs serve
-   
-Once you are happy with the changes, you can prepare a pull request to update the checked in docs as you would with any other code change. 
+1. Install python
+2. From this directory (\docs) run: pip install --upgrade -r requirements.txt
 
-When completed, then you will have to manually update the live docs by do the following from the root (todo: make better):
+## Viewing changes locally
+1. From this directory (\docs) run: mkdocs serve
 
-   1. (optional- in case you rev'd it) mkdocs build --clean --config-file .\docs\mkdocs.yml 
-   2. git checkout gh-pages
-   3. git checkout -b <topic branch with>
-   4. Commit your changes, open a pull request. Once approved and the remote 'gh-branch' has the last changes then it will update the website.
+## Commiting changes to the live site
+Once you are happy with your changes, you will have to manually update the live docs by doing the following from the root folder (TODO: make part of the CI build):
 
-## ## Open Source Software Acknowledgments
+   1. Checkout a working branch based on the latest code in 'main'.
+   2. Make your desired documentation changes.
+   3. mkdocs build --clean --config-file .\docs\mkdocs.yml
+   4. git checkout gh-pages
+   5. robocopy /S .\docs\site\ .\
+   6. git checkout -b <your working branch>
+   7. Commit & push your changes, open a pull request into 'gh-pages'. Once approved and the remote 'gh-pages' has the changes it will update the website automatically.

```

Comment:
i think we can enforce this by just limiting/requiring specific sign off for check in against the gh-pages to core team as to avoid unneeded churn. 

that is, i'd like to have an explicit 'public docs' sign off..

Reply:
I updated the gh-pages policies to require signoff before submit. Administrators (OneCore) must also follow this: https://github.com/microsoft/FactoryOrchestrator/settings/branch_protection_rules/18111886